core/src/edu/asu/c3simulator/screens/CouncilPanel.java

```diff
@@ -0,0 +1,46 @@
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+
+import edu.asu.c3simulator.widgets.NavigationPanel;
+
+/**
+ * Navigation Panel for council screens
+ * 
+ * @author Alyahya, Mohammed
```

Comment:
It isn't. Mohammed and I were pair programming a few days ago and just added all the code in share_branch_justin.